tsconfig.json

```
@@ -0,0 +1,4 @@
+{
+  "compilerOptions": {},
+  "extends": "expo/tsconfig.base"
+}
```

```suggestion
{
  "extends": "expo/tsconfig.base",
  "include": ["**/*.js", "**/*.jsx", "**/*.ts", "**/*.tsx"],
  "compilerOptions": {
    "strict": true,
    "skipLibCheck": true,
    "baseUrl": ".",
    "paths": {
      "@/*": ["*"]
    }
  }
}
```

Then you can update imports to use `@` alias